pandunia-lekse.csv

```diff
@@ -445,6 +448,7 @@ cape|||hin:छाप (chāp), ben:ছাপ (chap), may:cap, eng:chop|print (sta
 cargone||||square (tetragon)||cuadro||||||||||||||neliö|kwadrat (czworokąt, czworobok, tetragon)
 cari|||rus:четыре (chetyre), hye:չորս (čʿors), hin:mar:चार (cār), ben:চার (car), pan:ਚਾਰ (chār), guj:ચાર (cār), may:catur|four (4)|quatre (4)|cuatro (4)|quatro (4)|четыре (4)||四 (4)|四 (4)||||||||kvar (4)|neljä (4)|cztery (4)
 carme|||fra:por:charme, eng:charm, rus:шарм (šarm)|charm (attraction)|charme|encanto|charme|шарм||||||||||||vetovoima (šarmi)|czar (urok, wdzięk)
+catake|||hin:छत्रक (chatraka), ben:ছত্রাক (chôtrak), jpn:茸 (take)|mushroom|champignon|champiñón (hongo, seta)|cogumelo|гриб||蘑菇|茸|||कुकुरमुत्ता (खुंबी, छत्रक)||cendawan|uyoga||||
```

Comment:
I suppose this means the sporocarp i.e. the "fruiting body". So could it be "fungopale" (fruit of fungus)?

Reply:
that makes a lot of sense. I was trying to have a word specifically for fungi with stem, cap, and gills, but now that I think about it, that's too precise for a common root and it would probably be used more generally anyway.